draft-ietf-taps-interface.md

```diff
@@ -626,16 +631,14 @@ This property specifies whether an application considers it useful to
 transfer data across multiple paths between the same end hosts. Generally,
 in most cases, this will improve performance (e.g., achieve greater throughput).
 One possible side-effect is increased jitter, which may be problematic for
-delay-sensitive applications.
-
-The recommended default is to have this option.
+delay-sensitive applications. The recommended default is to Prefer this option.
 
 
 ### Direction of communication
 
```

Comment:
Despite this being a selection property, this is rather an Enum then a Preference. This was most probably lost on the pass that moved this property up in the document and removed the types.
As unidirectional applications can just ignore the other direction, using this as an Enum is safe.

Reply:
I'm fine with this staying a simple selection property. Having more granular control would be nice, but I'm not sure if we need to standardize that.

@philsbln 
By Enum you mean something like "Use second path as fallback", "aggregate bandwidth", or "do not use multiple paths"? Or are there more possible values?
I'm not sure what "the other direction" would mean here?

Reply:
I'm lost - this is about "Direction of communication", right? But it already has multiple values (bidirectional vs. unidirectional)?

Reply:
I thought this was about "Parallel Use of Multiple Paths".

Reply:
So I think that @philsbn was talking about "Direction of communication" but got confused. Then @theri thought it's about "Parallel Use of Multiple Paths". So altogether we're all confused, I think. Here's a suggestion: let me continue with this as-is on this PR, and fix it afterwards.

Reply:
Sorry – I was talking about "Direction of communication" … made the comment above the section.

Direction of communication is not a preference – It has three distinct values whereby one of these values can be used as an obvious fallback.

Reply:
Got it - fixed it by changing the text back (just removing the "Prefer").